pkgs/data/sgml+xml/schemas/xml-dtd/docbook/builder.sh

```diff
@@ -1,7 +1,8 @@
 source $stdenv/setup
 
-mkdir -p $out/xml/dtd/docbook
-cd $out/xml/dtd/docbook
+dir=$out/share/${stdenv.lib.replaceStrings [ "-xml" ] [ "" ] name}/dtd
```

Comment:
This would not work. You can use `${name/-xml/}` but people on IRC [recommend](https://botbot.me/freenode/nixos/2017-10-17/?msg=92400848&page=9) moving the builder to a Nix expression (`generic.nix`).

Reply:
I know - I was trying out some things but had to leave the computer I was working on, hence a commit named "fef". I don't like using ```builder.sh``` either - it's much easier to grok as part of the derivation directly.